### DISCUSSION TITLE
Welcome to audiotext Discussions!

### DISCUSSION THREAD

**Discussion by HenestrosaDev** [HenestrosaDev/audiotext#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rccnpcc-wq** [comment]
hey i  might sound stupid but i decompressed it but cannot find executible file. 
<img width="1639" height="1072" alt="image" src="https://github.com/user-attachments/assets/401111e9-7fd8-4b82-81d9-715a3db83de5" />

i downloaded the latest release.
